core/src/main/scala/scalaz/MonadError.scala

```diff
@@ -29,6 +29,25 @@ object MonadError {
   @inline def apply[F[_], S](implicit F: MonadError[F, S]): MonadError[F, S] = F
 
   ////
+  import Isomorphism.<~>
+
+  /**
+   * Derives a MonadError for something isomorphic to a thing with a MonadError.
+   */
+  def fromIsoWithMonadError[F[_], G[_], E](
+    D: F <~> G
+  )(
+    implicit ME: MonadError[G, E]
+  ): MonadError[F, E] = new MonadError[F, E] {
```

Comment:
https://github.com/scalaz/scalaz/blob/74f55de04e38cf82acb6f7ca55ec937c7ff6972d/core/src/main/scala/scalaz/Isomorphism.scala#L344-L345 ?

Reply:
lol, ok, that's nice. I didn't know these existed.

Reply:
it was all done in #722847f578be134f0182191444ce07ae45031e85

but I think my method is still useful as a convenience